.github/workflows/cypress.yml

```diff
@@ -27,7 +27,7 @@ jobs:
         ports:
           - 6379:6379
       ckan-solr:
-        image: solr:6.5
+        image: solr:8.0
```

Comment:
Recommend either `solr:8` or `solr:8.10`, as there's no point in pinning to an old minor release.

Reply:
So, `solr:8` will pin it to the latest v8 release, right? I prefer this option in that case

Reply:
Yes, currently, this is the (top of) the Solr docker tags:
![image](https://user-images.githubusercontent.com/36156267/142163839-2b33956d-dcd4-4261-84ec-02afc1586ff9.png)

So solr:8 will follow the latest solr 8.x.y release, where solr:8.0 would be solr 8.0.0:
![image](https://user-images.githubusercontent.com/36156267/142164003-62fcc21e-2a57-4064-a99a-f854b63adbc2.png)